src/components/Charts/ModelChart.style.js

```diff
@@ -120,14 +118,63 @@ export const Wrapper = styled.div`
 
   /* No action */
   .limited-action {
-    fill: ${props =>
-      props.projections.getChartSeriesColorMap().limitedActionSeries};
-    stroke: ${props =>
-      props.isInferred
-        ? props.projections.getChartSeriesColorMap().limitedActionSeries
-        : 'white'};
-    fill-opacity: 1;
+    .highcharts-graph {
+      stroke: ${COLORS.TRENDS_CONTINUE};
```

Comment:
Hrm.  Should this be called "LIMITED_ACTION" or "RESTRICTIONS_LIFTED" instead of "TRENDS_CONTINUE"?

Reply:
updated (`LIMITED_ACTION`)